.github/workflows/documentation.yml

```
@@ -8,13 +8,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "Checkout Instaloader Repository"
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0  # needed for building docs
       - name: "Setup Python"
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v5
         with:
           python-version: 3.12
+          cache: 'pip'
```

```suggestion
          cache: 'pipenv'
```

We actually use Pipenv for dependency management, Pip is only used in the Workflows to install Pipenv.

---

Thanks! I wasn't quite sure which one to use. Does that apply to "packages.yml" as well?

---

Thanks! Good point, "packages.yml" does acutally not install the Instaloader dependencies, it just installs the dependencies required for the packaging, and does so with Pip. However, the pip cache mechanism seems to be using the requirements.txt file which we do not have, so for packages.yml the cache should better be disabled.

---

Looks good, thank you very much!